### ISSUE TITLE
Dropdown: Default of isOpen=true leaves menu not keyboard accessible

### ISSUE THREAD

**Issue by jeff-phillips-18** [patternfly/patternfly-react#2083]
If the dropdown is programmatically set to isOpen=true, the drop down items are not reachable via keyboard navigation. 

If I click to open the dropdown, I cannot navigate to the items via the keyboard either.

Same is true for application launcher.

**Comment by karelhala** [comment]
Mhhh, this is interesting finding, however I would say that the issue is flipped. If you open dropdown or application launcher via keyboard you should cycle trough options with arrow keys. Once you reach end of dropdown and click on arrow down you select first option.

Honestly I find this more intuitive than cycling trough all options. Imagine having dropdown (or select) with 100 items, with tabs you'd have to go trough all items and after that you could select next component in row.

This is another example of different implementation of similiar components (dropdown vs app launcher vs select vs context select vs options select) and we should really focus on bringing things together with using one core component instead of copy code (addresses in https://github.com/patternfly/patternfly-react/issues/2065).

However I think that there is different kind of bug here, if you open dropdown with mouse or programatically first option is not selected and so you are not able to cycle trough them with arrows.

**Comment by jeff-phillips-18** [comment]
Yes @karelhala, that is the issue I was referring to. Once it is open other than via the keyboard, you cannot use the keyboard to access it.

**Comment by karelhala** [comment]
@jeff-phillips-18 oh okay gotcha, then we should olso point out that other dropdown-like (context selector, otpions menu and select) components are not using arrow keys for navigation.
